### PR TITLE
support staging mixer

### PR DESCRIPTION
**Pull request by hanlu09205**
Previous pull request in https://github.com/datacommonsorg/mixer/pull/95. 

Addressed Bo's comment regarding a new file for storing bt table instance. 

Then follow new pull request process and create this one.

### CODE REVIEW

File: deployment/gke.sh [hunk 1]
```diff
@@ -50,22 +50,42 @@ else
...
+  bt_table_input_file="./staging_bt_table.txt"
+fi
+while read -r line; do
+  BT_TABLE=$line
+done < "$bt_table_input_file"
+export BT_TABLE
+perl -i -pe's/BT_TABLE/$ENV{BT_TABLE}/g' deployment.yaml
 
-# Get a static ip address
+Get a static ip address
```
Comment (shifucun) [inline deployment/gke.sh]: need to add back these comments and lines below?

**Comment by hanlu09205** [comment]
> @googlebot I signed it!

@googlebot I signed it!

**Comment by hanlu09205** [comment]
> > @googlebot I signed it!
> 
> @googlebot I signed it!

**Comment by hanlu09205** [comment]
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
@googlebot I signed it!

**Pull request merged**